Resources/Locale/en-US/interaction/interaction-popup-component.ftl

```diff
@@ -4,6 +4,7 @@
 
 petting-success-generic = You pet {THE($target)} on {POSS-ADJ($target)} head.
 petting-success-soft-floofy = You pet {THE($target)} on {POSS-ADJ($target)} soft floofy head.
+petting-success-generic-others = { CAPITALIZE(THE($user)) } pets {THE($target)}.
```

Comment:
This is weird, petting your co-worker. In #563 consensus is that hugging someone is a little weird; petting is even more touching them randomly?

This can also be emoted.